src/test/java/lingogo/logic/commands/ImportCommandTest.java

```diff
@@ -55,47 +60,107 @@ public void execute_slideshowActive_throwsCommandException() {
 
     @Test
     public void execute_importInvalidHeaders_throwsCommandException() {
-        String filePath = "src/test/data/SampleCsvFiles/invalidHeaders.csv";
-        String expectedMessage = String.format(MESSAGE_INVALID_CSV_FORMAT, filePath);
-        ImportCommand command = new ImportCommand(filePath);
+        createDataFolder();
+        String fileName = "invalidHeaders.csv";
+        try {
+            createCopyInDataFolder(fileName);
+        } catch (Exception e) {
+            fail("Exception not expected");
+        }
+        String expectedMessage = String.format(MESSAGE_INVALID_CSV_HEADERS, fileName);
+        ImportCommand command = new ImportCommand(fileName);
         assertCommandFailure(command, model, expectedMessage);
+        try {
+            deleteCopyInDataFolder(fileName);
+        } catch (Exception e) {
+            fail("Exception not expected");
+        }
     }
 
     @Test
     public void execute_importInvalidContent_throwsCommandException() {
-        String filePath = "src/test/data/SampleCsvFiles/invalidContent.csv";
-        String expectedMessage = String.format(MESSAGE_INVALID_CSV_FORMAT, filePath);
-        ImportCommand command = new ImportCommand(filePath);
+        createDataFolder();
+        String fileName = "invalidContent.csv";
+        try {
+            createCopyInDataFolder(fileName);
+        } catch (Exception e) {
+            fail("Exception not expected");
+        }
+        String expectedMessage = String.format(MESSAGE_INVALID_CSV_CONTENT, fileName);
+        ImportCommand command = new ImportCommand(fileName);
         assertCommandFailure(command, model, expectedMessage);
+        try {
+            deleteCopyInDataFolder(fileName);
+        } catch (Exception e) {
+            fail("Exception not expected");
+        }
     }
 
     @Test
     public void execute_importDuplicateContent_noNewFlashcards() {
-        String filePath = "src/test/data/SampleCsvFiles/duplicateContent.csv";
-        String expectedMessage = String.format(ImportCommand.MESSAGE_SUCCESS, filePath);
-        ImportCommand command = new ImportCommand(filePath);
+        createDataFolder();
+        String fileName = "duplicateContent.csv";
+        try {
+            createCopyInDataFolder(fileName);
+        } catch (Exception e) {
+            fail("Exception not expected");
+        }
+        String expectedMessage = String.format(ImportCommand.MESSAGE_NOT_UPDATED, fileName);
+        ImportCommand command = new ImportCommand(fileName);
         try {
-            expectedModel.importFlashCards(filePath);
+            command.importHelper(expectedModel);
```

Comment:
We shouldn't need to do anything to `expectedModel` since it already has all the flashcards we expect right?